.github/workflows/js.yml

```
@@ -0,0 +1,29 @@
+      with:
+        fetch-depth: 2
+    - uses: actions/setup-node@v4
+      with:
+        node-version: '20'
+        cache: 'npm'
+    - name: Install npm dependencies
+      run: npm i --include=dev
+    - name: Static Tests
+      run: npm run static-test
+    - name: Live Tests
+      run: npm run test
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)